### ISSUE TITLE
Manticore 5 support

### ISSUE THREAD

**Issue by CGretski** [DEgITx/rats-search#176]
Bug in manticore searchd uses all available processor time.
[Upstream bug](https://github.com/manticoresoftware/manticoresearch/issues/578)

Please use version of manticore >= 4.0.2

**Comment by CGretski** [comment]
After making a copy of database folder  (to manticoredb) I have successfully used version 1.9.0 with a standalone instance of manticore 5.0.2 on Windows with no real problems.
`"d:\manticore\bin\searchd.exe" --config d:\rats\data\manticore.conf --pidfile`

There is an error in rats.log during startup, though it doesn't seem to affect anything:
[12:25:29] founded running sphinx instance in D:\rats/data/searchd.pid, using it
**[12:25:29] changed configuration user directory: undefined
[12:25:29] [ERROR] Rejection: TypeError: Error processing argument at index 1, conversion failure from undefined
    at App.<anonymous> (D:\rats\resources\app.asar\app\background.js:1:130641)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:5)**
[12:25:29] external sphinx signalled
[12:25:30] db version 8
[12:25:30] db patch done

My config file is attached:
[manticore.conf](https://github.com/DEgITx/rats-search/files/9692393/manticore.conf.txt)

Notable changes from original sphinx.conf:
- workers directive no longer exists
- configured empty "stored_fields" on torrents index as default for stored_fields is now all full-texts fields.

For performance tuning on my machine I also configured rt_mem_limit on the files and torrents indexes and disabled psuedo_sharding.
